### PR TITLE
fix for #123

### PR DESCRIPTION
**Pull request by Speedy1991**
fix the request context error (#123)
tested with:
```
>>> import django
>>> django.VERSION
(1, 11, 4, 'final', 0)
```

### CODE REVIEW

File: puput/templatetags/puput_tags.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
-from django.template import Library, loader
+from django.template import Library
 from django.core.urlresolvers import resolve
 
 from el_pagination.templatetags.el_pagination_tags import show_pages, paginate
+from django.template.loader import render_to_string
```
Comment (marctc) [inline puput/templatetags/puput_tags.py]: Can you place it under the line 3?

**Comment by marctc** [comment]
Amazing!!

**Comment by Speedy1991** [comment]
Thanks :)
i didn't notice there was already a `context `arg, renaed the local `context`
`context->ctx`

**Comment by marctc** [comment]
Its not the same context?

**Comment by Speedy1991** [comment]
as far i can see the context passed by args is an instance of `RequestContext`.
Django 11 don't process anymore RequestContext
See [this](https://docs.djangoproject.com/en/1.11/releases/1.11/#django-template-backends-django-template-render-prohibits-non-dict-context) for more information

**Pull request merged**